AltStoreCore/Model/AppPermission.swift

```diff
@@ -25,18 +25,44 @@ public extension ALTAppPermissionType
         switch self
         {
         case .photos: return NSLocalizedString("Photos", comment: "")
+        case .location: return NSLocalizedString("Location", comment: "")
+        case .contacts: return NSLocalizedString("Contacts", comment: "")
+        case .reminders: return NSLocalizedString("Reminders", comment: "")
+        case .appleMusic: return NSLocalizedString("Apple Music", comment: "")
+        case .microphone: return NSLocalizedString("Microphone", comment: "")
+        case .speechRecognition: return NSLocalizedString("Speech Recognition", comment: "")
         case .backgroundAudio: return NSLocalizedString("Background Audio", comment: "")
         case .backgroundFetch: return NSLocalizedString("Background Fetch", comment: "")
+        case .bluetooth: return NSLocalizedString("Bluetooth", comment: "")
+        case .network: return NSLocalizedString("Network", comment: "")
+        case .calendars: return NSLocalizedString("Calendars", comment: "")
+        case .touchID: return NSLocalizedString("Touch ID", comment: "")
+        case .faceID: return NSLocalizedString("Face ID", comment: "")
+        case .siri: return NSLocalizedString("Siri", comment: "")
```

Comment:
The Xcode signer won't sign apps with Siri entitlements. Are you sure that SideStore can resign apps with this?

Reply:
Doesn't hurt to have them in there for now in case that ever changes. 

The code should strip that entitlement in the prepare stage if not already. 

Not even sure if that's possible, but still, having the key/value mappings here is harmless regardless I'm assuming?

Reply:
We could also extend the enum with a 'var isSupported: Bool { switch self...}'